src/ExpressionEvaluator/Package/ExpressionEvaluatorPackage.csproj

```diff
@@ -50,9 +50,9 @@
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
```

Comment:
Correct. 

I also have no idea how this file got changed.  I never even opened it?!